config/neo4j.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 data:
-  NEO4J_URI: "bolt://neo4j:7687"
+  NEO4J_URI: "bolt://neo4j.staging:7687"
```

Comment:
This also would break I think

Reply:
No, that's default kubernetes ~behaviour~ magic. You can check yourself by
```
kubectl --namespace=staging exec -it <POD_ID> nslookup <SERVICE_NAME>.staging
```
The trailing `staging` is optional, if you omit it, it will match all services in all namespaces, as far as I know.

See here: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#a-records